_layouts/dataset.html

```diff
@@ -15,11 +15,14 @@
 <div class="row" data-hook="read-view">
   {% if organization %}
   <div class="col-sm-3">
+    <div class="panel panel-default">
     {% if organization.logo %}
       <a href="{{ site.baseurl }}/datasets/?organization={{ organization.title | slugify }}" class="thumbnail"><img src="{{ organization.logo }}" alt="{{ organization.title }} logo"></a>
     {% endif %}
-    <h3><a href="{{ site.baseurl }}/datasets/?organization={{ organization.title | slugify }}">{{ organization.title }}</a></h3>
-    <p>{{ organization.description }}</p>
+      <div class="panel-heading"><h3><a href="{{ site.baseurl }}/datasets/?organization={{ organization.title | slugify }}">{{ organization.title }}</a></h3>
+        <div class="panel-body">{{ organization.description }}</p></div>
```

Comment:
Good idea putting this in a panel. But I think you've accidentally put the `panel-body` inside the `panel-heading` rather than as a sibling. Also, I think we should add a conditional to test whether there is a description set, otherwise there's an empty box.